src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/service/ExternalUsersApiService.kt

```diff
@@ -128,6 +129,13 @@ class ExternalUsersApiService(
       .bodyToMono(GroupDetails::class.java)
       .block(timeout) ?: throw GroupNotFoundException("get", group, "notfound")
 
+  fun getChildGroupDetail(group: String): ChildGroupDetails =
+    externalUsersWebClient.get()
+      .uri("/groups/child/$group")
+      .retrieve()
+      .bodyToMono(ChildGroupDetails::class.java)
+      .block(timeout) ?: throw RuntimeException("Failed to retrieve child group details")
+
```

Comment:
Prob worth checking with Ravi what he's doing with the other timeouts for HAAR-931

Reply:
Discussed, Ravi to test & feed back under his ticket